src/metabase/pulse/render/table.clj

```diff
@@ -13,24 +14,22 @@
    (style/font-style)
    {:font-size :12px
     :font-weight     700
-    :color           style/color-text-medium
-    :border-bottom   (str "1px solid " style/color-header-row-border)
-    :padding-top     :20px
-    :padding-bottom  :5px}))
+    :color           style/color-text-dark
+    :border-bottom   (str "2px solid " style/color-border)
+    :border-right    0}))
```

Comment:
this is a style change for the table header, table header style inherit the style from table body and extends/overrides some properties.

As you can see from the design, we don't want vertical border between each headers, but I include that in the table body style, that's why I need to override `border-right` here.

Also, the `border-bottom` on the header is a bit thicker than other border (2px vs 1px).

![image](https://user-images.githubusercontent.com/1937582/190602821-61ba291c-df11-4679-a3d3-0574dfca2410.png)